.travis.yml

```diff
@@ -6,8 +6,16 @@ cache: bundler
 sudo: false
 bundler_args: --without local
 rvm:
-- 2.1.8
+  - 2.1.8
+  - 2.2.5
```

Comment:
we should update this to 2.3.1, that's what chef 12.14 packages as the embedded ruby.